mlflow/projects/kubernetes.py

```diff
@@ -119,7 +119,7 @@ def wait(self):
 
```

Comment:
@harupy Do you know why we create `kube_api` client every time when calling the method ? Shouldn't it being created in the __init__ ?